src/main/java/spotify/playback/data/lyrics/GeniusLyricsScraper.java

```diff
@@ -1,8 +1,12 @@
 package spotify.playback.data.lyrics;
```

Comment:
Overall, I really love these changes! It would be cool to have some proper unit tests though. If you could do me a favor, could you send me a list of a couple songs with problematic characters?

Reply:
Songs that didnt work before:
https://open.spotify.com/track/4zdQmfTLWgGd5mAX4MUIaX?si=5c4b85b0ad99471d
https://open.spotify.com/track/4bV5sf2B4hWBBd5HQ8S7KB?si=d3ebbda2aceb4a4c
https://open.spotify.com/track/5Hijdt7rmbj9fUJdXEs6Nz?si=70771352bc7d4ef6
https://open.spotify.com/track/5KI4XeWRCHCFwkL2ozf06m?si=5345c2aeb98846ef
https://open.spotify.com/track/4E23uX1BDdUTk9x56nEbla?si=4f412e76206a4690
https://open.spotify.com/track/4IMP7v2UPYtFqjWucd7Lfw?si=aa4f600a978141bb
https://open.spotify.com/track/4HMop4Re0iucehmF7mgV27?si=b9d333fd10ec4a5f

Songs that still dont work:
https://open.spotify.com/track/23RoR84KodL5HWvUTneQ1w?si=0f2e2b0e33224178
     https://genius.com/Peggy-gou-it-goes-like-nanana-edit-lyrics
https://open.spotify.com/track/6q6hG6t5CWF501VRpBT1gC?si=b5df678e216f4639
     https://genius.com/2-2mashi-barefoot-lyrics
https://open.spotify.com/track/5gEEcgxnyagVvAZlQ43dfn?si=43395de456ae4ecf
     https://genius.com/Tk-from-unravel-lyrics
     
Notes. lots of remixs and edits dont work but the original version does. The only option would be to get the original lyrics but they likely wouldn t match the remix/edit